substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java

```diff
@@ -75,6 +75,10 @@
     @Option(help = "Build statically linked executable (requires static libc and zlib)")//
     public static final HostedOptionKey<Boolean> StaticExecutable = new HostedOptionKey<>(false);
 
+    @APIOption(name = "target")//
+    @Option(help = "Selects native-image compilation target (in <OS>-<architecture> format)")//
+    public static final HostedOptionKey<String> TargetPlatform = new HostedOptionKey<>(System.getProperty("os.name") + "-" + System.getProperty("os.arch"));
```

Comment:
I thought that you say in the message what is the default. And the value should stay as "".